multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ScaleAppStep.java

```diff
@@ -27,7 +28,12 @@ protected StepPhase executeStep(ProcessContext context) {
         CloudControllerClient client = context.getControllerClient();
 
         String appName = app.getName();
-        Integer instances = (app.getInstances() != 0) ? app.getInstances() : null;
+
+        if (app.getInstances() == 0) {
+            throw new SLException(Messages.ERROR_INVALID_ZERO_INSTANCE_COUNT);
+        }
+
+        Integer instances = app.getInstances();
```

Comment:
I think you can leave this line before if statement because you are using it there